shared/build.gradle.kts

```diff
@@ -7,6 +7,10 @@ android {
     namespace = "kmp.shared"
 }
 
+multiplatformResources {
+    multiplatformResourcesPackage = "kmp.shared"
```

Comment:
Maybe we should also turn off the warning about having to copy resources to the static framework, so we don't have nonsense in the build logs